app/Http/Controllers/CommentsController.php

```diff
@@ -116,17 +117,16 @@ public function index()
         } else {
             $commentBundle->depth = 0;
             $commentBundle->includeCommentableMeta = true;
-            $commentBundle->includeDeleted = isset($commentable);
-            $commentBundle->includePinned = isset($commentable);
+            $commentBundle->includePinned = false;
 
             $commentPagination = new LengthAwarePaginator(
                 [],
-                Comment::count(),
+                $commentBundle->countForPaginator(),
                 $commentBundle->params->limit,
                 $commentBundle->params->page,
                 [
                     'path' => LengthAwarePaginator::resolveCurrentPath(),
-                    'query' => $commentBundle->params->forUrl(),
+                    'query' => array_merge($commentBundle->params->forUrl(), ['commentable_id' => $id, 'commentable_type' => $type]),
```

Comment:
should those just be part of `forUrl`?

Reply:
it seemed odd to have it as part of `forUrl` when `CommentBundleParams` didn't do anything with them 🤷‍♂